mod/wikirate_source/set/type/source/source.rb

```diff
@@ -131,23 +151,24 @@ def parse_source_page url
 
 event :autopopulate_website, :after=>:approve_subcards, :on=>:create do
   website = Card[:wikirate_website].name
-  if link_card = subcards["+#{ Card[:wikirate_link].name }"] and link_card.errors.empty?
-    website_subcard = subcards["+#{website}"]
+  if link_card = subfield(:wikirate_link) and link_card.errors.empty?
+    website_subcard = subfield(website)
     unless website_subcard
       host = link_card.instance_variable_get '@host'
       website_card = Card.new :name=>"+#{website}", :content => "[[#{host}]]", :supercard=>self
       website_card.approve
-      subcards["+#{website}"] = website_card
+      # subcards["+#{website}"] = website_card
+      add_subcard "+#{website}", website_card
       if !Card.exists? host
         Card.create :name=>host, :type_id=>Card::WikirateWebsiteID
       end
     end
   end
-  if subcards["+File"]
+  if subfield('File')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.